assets_management/tests/test_assets_management.py

```diff
@@ -145,26 +145,3 @@ def _get_report(self, report_date, report_type):
         report_ids = report_result['context']['active_ids']
         report = self.env[report_model].browse(report_ids)
         return report
-
-    def test_journal_prev_year(self):
```

Comment:
Puoi chiarire quale sarebbe il problema con questo test?
Normalmente se un test fallisce vuol dire che c'è un problema ed eliminare il test non risolve il problema.

Per aggiungere un po' di contesto, il test fu aggiunto per https://github.com/OCA/l10n-italy/pull/3010.

Reply:
> Puoi chiarire quale sarebbe il problema con questo test? Normalmente se un test fallisce vuol dire che c'è un problema ed eliminare il test non risolve il problema.

Sono perfettamente d'accordo.
In questo specifico caso avevo dato priorità al fatto che fosse un problema bloccante per tutte le PR attualmente aperte. Quindi l'intenzione era di procedere al merge di questa come hotfix e poi risolvere con calma il problema del test riaggiungendolo con una successiva PR.

> 
> Per aggiungere un po' di contesto, il test fu aggiunto per #3010.

Ah ecco, grazie mille mi era sfuggita!
Avevo dato un'occhiata (troppo) veloce alla PR di migrazione della 14.0 e avevo visto che il test non era presente.

Adesso mi è tutto più chiaro e dovrei aver capito qual'è l'inghippo.
Aggiorno la PR.

Reply:
>  Aggiorno la PR.

Fatto.